src/sentry/static/sentry/less/group-detail.less

```diff
@@ -1410,7 +1410,6 @@ span.val {
   color: darken(@purple, 10);
   border-radius: 3px;
   padding: 2px 4px;
-  margin: -2px -4px;
```

Comment:
Removed this as it was causing styling issues. Introduced in https://github.com/getsentry/sentry/commit/0cd9514b432943795da44256fc065f17762b2f32



Before: 
<img width="1308" alt="Screen Shot 2020-06-24 at 4 54 04 AM" src="https://user-images.githubusercontent.com/139499/85525410-100b6600-b5d7-11ea-817f-ae8738543433.png">

After:

<img width="1306" alt="Screen Shot 2020-06-24 at 4 54 32 AM" src="https://user-images.githubusercontent.com/139499/85525407-0f72cf80-b5d7-11ea-9cd0-b4d377396dd6.png">

Reply:
Should we dial down the horizontal padding in the value elements? It seems wider than before.

Reply:
@markstory The padding hasn't changed. I prefer that to be updated in another PR as that's out of scope.